ansible/tasks/static-sticky.yml

```diff
@@ -0,0 +1,99 @@
+---
+
+- name: "Add user static-sticky"
+  user:
+    name: "static-sticky"
+    shell: "/usr/sbin/nologin"
+    home: "/var/www/static-sticky"
+    system: true
+
+- name: "copy systemd unit"
+  template:
+    src: "templates{{ item }}.j2"
+    dest: "{{ item }}"
+  with_items:
+    - "/etc/systemd/system/deploy-static-sticky.service"
+
+- name: "ensure polkit rules dir exists"
+  file:
+    path: "/etc/polit-1/rules.d"
+    state: "directory"
+
+- name: "copy polkit rules"
+  template:
+    src: "templates{{ item }}.j2"
+    dest: "{{ item }}"
+  with_items:
+    - "/etc/polkit-1/rules.d/00-deploy-static-sticky-service.rules"
+
+- block:
+  - name: "copy deploy script"
+    template:
+      src: "templates{{ item }}.j2"
+      dest: "{{ item }}"
+      mode: "u+rwx"
+    with_items:
+      - "/var/www/static-sticky/deploy_static_sticky.sh"
+
+  - name:
+      "clone static-sticky repository"
+    git:
+      repo: "https://github.com/svsticky/static-sticky.git"
+      dest: "/var/www/static-sticky/static-sticky"
+      version: "master"
+      depth: 1
+      force: true
+    diff: false
+
+  - name:
+      "template environment file for static-sticky"
+    template:
+      src: "templates{{ item }}.j2"
+      dest: "{{ item }}"
+    diff: false
+    with_items:
+      - "/var/www/static-sticky/static-sticky/.env"
+
+  - name: "clone NVM"
+    git:
+      repo: "https://github.com/creationix/nvm.git"
+      dest: "/var/www/static-sticky/.nvm"
+      depth: 1
+    diff: false
+
+  - name: "install node version"
+    shell: |
+      . /var/www/static-sticky/.nvm/nvm.sh
+      nvm install
+    args:
+      chdir: "/var/www/static-sticky/static-sticky"
+      executable: "/bin/bash"
+
+  - name: "run npm install"
+    shell: |
+      . /var/www/static-sticky/.nvm/nvm.sh
+      npm install
+    args:
+      chdir: "/var/www/static-sticky/static-sticky"
+      executable: "/bin/bash"
+
+  - name: "build website"
```

Comment:
not exactly, there could also be changes in contentful

Reply:
Yes, but I was under the impression that we'd use the webhook for triggering on Contentful changes, and not a manual Ansible run, since we really don't want the board to have to come running to us to deploy every small content change.

I know the Contentful webhook implementation isn't done yet, but I think we _will_ need a basic implementation of that (not in this PR though), before taking the website to production. So my comment was basically anticipating that situation.